pkgs/applications/system/coolercontrol/coolercontrol-ui-data.nix

```diff
@@ -1,17 +1,25 @@
-{ buildNpmPackage
-}:
+{ buildNpmPackage, autoPatchelfHook }:
 
-{ version
-, src
-, meta
+{
+  version,
+  src,
+  meta,
 }:
 
 buildNpmPackage {
   pname = "coolercontrol-ui";
   inherit version src;
   sourceRoot = "${src.name}/coolercontrol-ui";
 
-  npmDepsHash = "sha256-PpX9lk+yEG1auvBv5JBdMh7rjWoM0oTYJx6Nme5ij8s=";
+  npmDepsHash = "sha256-j+bGOGIG9H/1z0dN8BfvWSi6gPvYmCV7l0ZNH8h3yeU=";
+
+  preBuild = ''
+    autoPatchelf node_modules/sass-embedded-linux-x64/dart-sass/src/dart
```

Comment:
Is this a temporary or permanently needed patch?

Reply:
Permanently needed, as `sass-embedded` includes a pre-compiled binary that requires patching to run.

> This package is an alternative to the [`sass`](https://www.npmjs.com/package/sass) package. It supports the same JS API as `sass` and is maintained by the same team, but where the `sass` package is pure JavaScript, `sass-embedded` is instead a JavaScript **wrapper around a native Dart executable**.

(Emphasis mine)

Reply:
Ok. This was changed to `sass-embedded` because of some npm warnings and this was the official recommended solution, in addition to some other config changes. I'll keep this is mind to see if we can get rid of it in the future though.